checkout/other-features.md

```diff
@@ -1290,15 +1289,30 @@ although that might be possible in the future.
 
 {% include expand-parameter.md %}
 
-## Payee Info
-
 {% include payee-info.md payment_instrument="invoice" %}
 
 {% include merchant-authenticated-consumer.md %}
 
 {% include settlement-reconciliation.md %}
 
-{% include iterator.html prev_href="summary" prev_title="Back: Summary" %}
+## Updating Payment Menu
+
+When the contents of the shopping cart changes or anything else that affects
+the amount occurs, the `paymentorder` must be updated and the Payment Menu
+must be `refresh`ed.
+
+{% include alert.html type="neutral" icon="info" body= "Features that are not
+described in the previous sections must not be used, although they are
+available in the API. Flags that can be turned to `true` must be kept
+`false` as described in this standard setup documentation." %}
+
+{% include alert.html type="neutral" icon="info" body= "Your integration must be
+resilient to change. Properties, operations, headers, etc., that aren't
+understood in any response **must be ignored**. Failing due to a something
+occurring in a response that your implementation haven't seen before is a
+malfunction that must be fixed." %}
```

Comment:
I believe the language of this alert was softened at some point. If not, let's try to soften it now.